### PR TITLE
[docs] Update the EAS update develop faster doc

### PR DESCRIPTION
**Pull request by wschurman**
# Why

Noticed that this doc had some stale information that was no longer applicable after removal of classic updates.

Also noticed that it was very particular to iOS internal distribution builds but didn't specify that. So I just removed a lot of the specifics there (since there are better docs for that) and kept a summary.

While this doc now is somewhat short and maybe not useful, it's no longer incorrect. Happy to just delete it if others think that's better.

# Test Plan

Inspect.

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/eas-update/develop-faster.mdx [hunk 1]
```diff
@@ -1,42 +1,20 @@
...
-
-<ImageSpotlight
-  alt="Preview EAS Update QR Code configuration"
-  src="/static/images/eas-update/qr-preview.png"
-  style={{ maxHeight: '700px', width: 'auto' }}
-/>
-
-For more information on using EAS Update with development builds, see [Using expo-dev-client with EAS Update](/eas-update/expo-dev-client).
+A published update can be loaded in various environments:
+- An [expo-dev-client](/eas-update/expo-dev-client/) build with a matching runtime version. Either an [internal distribution](/build/internal-distribution/) or simulator/emulator build.
```
Comment (brentvatne) [inline docs/pages/eas-update/develop-faster.mdx:18]:
```suggestion
- A [development build](/eas-update/expo-dev-client/) with a matching runtime version, distributed through either [internal distribution](/build/internal-distribution/) or as a simulator/emulator build.
```

File: docs/pages/eas-update/develop-faster.mdx [hunk 2]
```diff
@@ -1,42 +1,20 @@
...
+- A build distributed via TestFlight, Google Play beta, or [internal distribution](/build/internal-distribution/) with a matching runtime version and channel pointing to a branch with the published update. With this strategy, often a EAS Update channel like "alpha" or "beta" will be used, and builds and updates will be created from development branches in source control.
```
Comment (brentvatne) [inline docs/pages/eas-update/develop-faster.mdx:19]:
```suggestion
- A release build distributed via TestFlight, Google Play beta, or [internal distribution](/build/internal-distribution/) with a matching runtime version and channel pointing to a branch with the published update. With this strategy, often a EAS Update channel like "alpha" or "beta" will be used, and builds and updates will be created from development branches in source control.
```

**Review by brentvatne — APPROVED** [review]

File: docs/pages/eas-update/develop-faster.mdx [hunk 3]
```diff
@@ -1,42 +1,20 @@
...
 
-## Developing locally
+## Distributing updates to others during development
 
-When developing locally, `npx expo start` will start a server that serves a manifest and assets to Expo Go and development builds. Locally, those builds look for a local manifest, and when there's an update, they'll download any missing local assets. To accomplish this, Expo Go/development builds use a protocol. That protocol is either a classic version of the updates protocol (used with Classic Updates, now deprecated), or the modern version of the [updates protocol](/technical-specs/expo-updates-1) (used with EAS Update).
+Whether a new feature is being developed, stabilized, QA'd, or critiqued, it's often useful to to allow others to preview what it looks and feels like before it is released to production.
 
-We want to make sure that our locally developed app is using the same protocol locally as it does when we build our app.
+Traditionally previewing meant re-building the project distributing it to a reviewer manually. Previewing features with EAS Update removes the need for re-building in most cases and can help us preview new features more quickly.
```
Comment (quinlanj) [inline docs/pages/eas-update/develop-faster.mdx:15]:
```suggestion
Traditionally previewing meant re-building the project and distributing it to a reviewer manually. Previewing features with EAS Update removes the need for re-building in most cases and can help us preview new features more quickly.
```

**Review by quinlanj — APPROVED** [review]

**Review by amandeepmittal — APPROVED** [review]

File: docs/pages/eas-update/develop-faster.mdx (hunk 2 above)
Comment (amandeepmittal) [reply docs/pages/eas-update/develop-faster.mdx:19]: I made one edit to Brent's suggestion to prefer mentioning Google Play Beta before TestFlight (as per our writing [style guide's platform order](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web)).

File: docs/pages/eas-update/develop-faster.mdx (hunk 1 above)
Comment (amandeepmittal) [reply docs/pages/eas-update/develop-faster.mdx:18]: Made on edit to Brent's suggestion to update `an emulator or simulator`.

**Pull request merged**